services/order.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from django.db import transaction
+
+from db.models import Ticket, User, Order, MovieSession
+
+
+def create_order(
+        tickets: List[dict],
+        username: str,
+        date=None
+):
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
Instead of User better use get_user_model (best practice)
Ex:
`user = get_user_model().objects.get(username=username)`